requirements/base.txt

```diff
@@ -1009,7 +1005,7 @@ psutil==5.9.8 \
     --hash=sha256:d06016f7f8625a1825ba3732081d77c94589dca78b7a3fc072194851e88461a4 \
     --hash=sha256:d16bbddf0693323b8c6123dd804100241da461e41d6e332fb0ba6058f630f8c8
     # via py7zr
-psycopg[binary]==3.1.10 \
+psycopg==3.1.10 \
```

Comment:
:raised_eyebrow: ?

Reply:
De ce que je comprend, `uv` a choisi de supprimer l'extra vu qu'au final tout est aplatis et qu'en le laissant on peux se retrouver à installer des choses non prévues mais aussi avoir des problèmes dans certain cas, mais il y a une option si besoin : astral-sh/uv/pull/2555.
Sachant que c'est déjà une discussion présente du coté de pip-tools (https://github.com/jazzband/pip-tools/issues/1613) ou de pip pour le fichier de contrainte (https://github.com/pypa/pip/issues/11599).

`psycopg-binary` étant l'extra derrière `[binary]` et étant bien présent je pencherais de ne pas avoir l'extra dans les `.txt` et seulement dans le `.in`.

Reply:
:+1: Merci pour l’explication !